### PR TITLE
[DST-65]: Adding Space for Popover based on theme

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

**Comment by OsamaAbdellateef** [comment]
@sebald 
There are two solutions for me 
- first is to use `useTheme` hook and based on the current theme we set the space 
- second is to create styles files for popover in `core/b2b-theme` which will be a good base to build on it

**Comment by sebald** [comment]
@OsamaAbdellateef yeah I think the second one makes the most sense, right?

**Comment by OsamaAbdellateef** [comment]
> @OsamaAbdellateef yeah I think the second one makes the most sense, right?

yeah right

File: packages/components/src/Overlay/Popover.test.tsx [hunk 1]
```diff
@@ -13,6 +13,13 @@ import { setup } from '../test.utils';
 const theme: Theme = {
   name: 'test',
   components: {
+    Popover: cva(['mt-0.5'], {
```
Comment (sebald) [inline packages/components/src/Overlay/Popover.test.tsx:16]: Hm, I think we always need to consider the position, right? The placement has to be passed to the styles inside the component.

**Review by sebald — CHANGES_REQUESTED** [review]

File: packages/components/src/Overlay/Popover.test.tsx (hunk 1 above)
Comment (OsamaAbdellateef) [reply packages/components/src/Overlay/Popover.test.tsx:16]: @sebald  
then should i create a component and create test for it  ?

Comment (OsamaAbdellateef) [reply packages/components/src/Overlay/Popover.test.tsx:16]: @sebald 
I don't understand

Comment (sarahgm) [reply packages/components/src/Overlay/Popover.test.tsx:16]: I also don't understand @sebald

Comment (sarahgm) [reply packages/components/src/Overlay/Popover.test.tsx:16]: I thought this issue is about to make a space prop where you can put in any value you want 🤔

Comment (sebald) [reply packages/components/src/Overlay/Popover.test.tsx:16]: Yes, but adding matching to the top will not work when the popover is shown above the anchor (e g. when there is not enough space before the anchor)

**Review by sebald — DISMISSED** [review]

**Review by sebald — APPROVED** [review]

**Pull request merged**